pypy/install

```diff
@@ -13,4 +15,10 @@ apt-get update
 apt-get install pypy pypy-dev git -y
 rm -f /usr/bin/python
 ln -s /usr/bin/pypy /usr/bin/python
-curl -sL https://raw.github.com/pypa/pip/master/contrib/get-pip.py | pypy
+curl -sL https://raw.github.com/pypa/pip/master/contrib/get-pip.py | python
```

Comment:
Running it locally I get a error:
"You're using an outdated location for the get-pip.py script, please use the one available from https://bootstrap.pypa.io/get-pip.py"